docs/example.md

```diff
@@ -54,9 +54,9 @@ You should see a collection of virtual services, virtual nodes and a mesh, along
 
 Verify App Mesh is working
 ```
-kubectl run -n appmesh-demo -it curler --image=tutum/curl /bin/bash
+kubectl run -n appmesh-demo -it curler --image=pstauffer/curl sh
```

Comment:
How about using the official curl image from the folks behind cURL?

https://hub.docker.com/r/curlimages/curl

Reply:
Agreed. Have made the changes.